plugins/kubernetes/test/controllers/kubernetes_dashboard_controller_test.rb

```diff
@@ -1,34 +1,42 @@
 require_relative '../test_helper'
 
 describe KubernetesDashboardController do
+  let(:project) { projects(:test) }
+  let(:environment) { environments(:production) }
+  let(:release) { kubernetes_releases(:test_release) }
+
   as_a_viewer do
     before do
       Kubernetes::Cluster.any_instance.stubs(:client).returns(Kubeclient::Client.new('http://cluster.localhost'))
-      stub_request(:get, %r{http://cluster\.localhost.*}).to_return(body: '{}')
     end
 
     it 'returns an empty array if no pods are running' do
-      get :index, project_id: :foo, environment: environments(:production).id, format: :json
+      environment.cluster_deploy_groups.each do |cluster_deploy_group|
+        stub_request(:get, %r{http://cluster\.localhost.*/namespaces/#{cluster_deploy_group.namespace}.*})
+          .to_return(body: '{}')
```

Comment:
Can't remember, I think I had problems with this and had to stub one request for each namespace. Can try to change it back and see if it still works.